CODEOWNERS

```diff
@@ -0,0 +1 @@
+* jfrerich iomodo
```

Comment:
I think these need the `@` prefix.  

Here a merged ticket with them added and shows the result of the new CODEOWNER FILES :) Notice the shield.


![image](https://user-images.githubusercontent.com/7575921/78717821-d3d53d00-78e6-11ea-8e6e-fd745205b365.png)

Reply:
Good catch :+1: